### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by ninariccimarie**
issue: https://github.com/ninariccimarie/quiz-master/issues/4

- [x] Setup I18n for the backend app
- [x] Update en.yml and introduce en-PH.yml
- [x] Remove all hardcoded strings inside our codebase and display the messages by using I18n.t

### CODE REVIEW

**Comment by ninariccimarie** [comment]
@rbmrclo done todos from[ issue Support I18n for PH #4](https://github.com/ninariccimarie/quiz-master/issues/4)

File: backend/config/initializers/locale.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+I18n.available_locales = [:en, :"en-PH"]
```
Comment (rbmrclo) [inline backend/config/initializers/locale.rb]: Please move this in `config/application.rb`. You can see the guide [here](https://github.com/rails/rails/blob/df94b863c2ff8f1bcf12e36ed8fc1419292668e7/guides/source/configuring.md#configuring-i18n)

File: backend/config/locales/en-PH.yml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+en-PH:
+  correct: "Yehey! Tama ka!"
+  wrong: "Pasensya, mali and iyong sagot. :("
```
Comment (rbmrclo) [inline backend/config/locales/en-PH.yml]: Lol! 😅

File: backend/Gemfile [hunk 3]
```diff
@@ -15,6 +15,7 @@ gem 'turbolinks', '~> 5'
 gem 'jbuilder', '~> 2.5'
 gem 'numbers_in_words'
 gem 'rack-cors'
+gem 'rails_admin', '~> 1.2'
```
Comment (rbmrclo) [inline backend/Gemfile]: Please remove. Thanks!

File: backend/config/initializers/locale.rb (hunk 1 above)
Comment (ninariccimarie) [inline backend/config/initializers/locale.rb]: [done](https://github.com/ninariccimarie/quiz-master/pull/8/commits/be8d3eb03e280e3507aed899188fa1b52230f4d8)

also tested 👍

File: backend/config/application.rb [hunk 4]
```diff
@@ -8,6 +8,13 @@
 
 module QuizMaster
   class Application < Rails::Application
+
+    config.i18n.fallbacks = true
+    config.i18n.fallbacks = [:en, :"en-PH"]
+
+    def default_url_options
+      { locale: I18n.locale }
+    end
```
Comment (rbmrclo) [inline backend/config/application.rb]: This method does nothing, let's just remove it!

Comment (ninariccimarie) [inline backend/config/application.rb]: [done](https://github.com/ninariccimarie/quiz-master/pull/8/commits/1552fe6c88442757d301786fed6a4ee2695a7192) removing. Please check @rbmrclo

**Comment by rbmrclo** [comment]
LGTM 👍  Could you remove `[WIP]` so I can already merge? @ninariccimarie

**Pull request merged**